cosmos-tx/Cargo.toml

```diff
@@ -11,7 +11,7 @@ categories = ["cryptography", "cryptography::cryptocurrencies", "encoding"]
 keywords   = ["blockchain", "cosmos", "tendermint", "transaction"]
 
 [dependencies]
-cosmos-sdk-proto = { version = "0.3", default-features = false, path = "../cosmos-sdk-proto" }
+cosmos-sdk-proto = { default-features = false, path = "../cosmos-sdk-proto" }
```

Comment:
Without explicitly versioned dependencies, it's not possible to publish crates to https://crates.io

What problem was this causing?

Reply:
I see, I just had to update it by hand and didn't see it as a good use of time.

Never tried to publish crates from a workspace before, usually break the crates out into their own repos. will correct this real quickly. 

Thanks!